plugins/plugin-testing-java/plugin-testing-junit/che-plugin-testing-junit-ide/src/main/java/org/eclipse/che/plugin/testing/junit/ide/action/JUnitTestActionGroup.java

```diff
@@ -11,17 +11,17 @@
 package org.eclipse.che.plugin.testing.junit.ide.action;
 
 import com.google.inject.Inject;
-import org.eclipse.che.ide.api.action.Action;
 import org.eclipse.che.ide.api.action.ActionManager;
+import org.eclipse.che.ide.api.action.BaseAction;
 import org.eclipse.che.ide.api.action.DefaultActionGroup;
 import org.eclipse.che.plugin.testing.ide.action.TestAction;
 
 /** Registrar of JUnit run/debug actions. */
 public class JUnitTestActionGroup implements TestAction {
   public static final String TEST_ACTION_RUN = "TestJUnitActionRun";
   public static final String TEST_ACTION_DEBUG = "TestJUnitActionDebug";
-  private final Action runTestAction;
-  private final Action debugTestAction;
+  private final BaseAction runTestAction;
```

Comment:
question: why do we require BaseAction there ?

Reply:
its refactoring artifact, I will replace it with Action